tests/richenum/test_ordered_rich_enums.py

```diff
@@ -45,8 +45,9 @@ def test_lookup_by_index(self):
         # Should work if enum isn't zero-indexed.
         self.assertEqual(SadBreakfast.from_index(1), oatmeal)
 
-        with self.assertRaises(EnumLookupError):
+        with self.assertRaises(EnumLookupError) as error:
             SadBreakfast.from_index(7)
+        self.assertTrue(isinstance(error.exception, LookupError))
```

Comment:
This should be it's own test. e.g. a test for exception inheritance. Then we don't need to add addition assert checks in these unrelated tests.